src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx

```diff
@@ -494,9 +622,28 @@ class ExplorerButtonGroup extends React.Component {
         || this.state.exportingToSevenBridges
         || this.isPFBRunning());
     }
+    if (buttonConfig.type === 'export-files') {
+      if (!this.props.buttonConfig.terraExportURL) {
+        console.error('Export to Terra button is present, but there is no `terraExportURL` specified in the portal config. Disabling the export to Terra button.'); // eslint-disable-line no-console
```

Comment:
something worth noting: each re-render (like clicking a checkbox) produces this error print 12 times :O

Reply:
if it's too much work, no worries, but -- the constructor might be a better place to perform these config checks

Reply:
Ya good idea, I just stuck these checks in the constructor!